securedrop/tests/functional/source_navigation_steps.py

```diff
@@ -47,7 +47,7 @@ def _source_clicks_submit_documents_on_homepage(self):
         assert self._is_on_generate_page()
 
     def _source_regenerates_codename(self):
-        self.safe_click_by_id("regenerate-submit")
+        self.driver.refresh()
```

Comment:
From my read, `test_refreshed_duplicate_generate_pages` is still useful even without the "refresh codename" feature (for example, it tests that the user is redirected to the lookup page when they have already logged in in a separate tab), so I have simply altered the behavior here to reload the page instead.